### ISSUE TITLE
Cannot rotate, mirror or scale in 15.06

### ISSUE THREAD

**Issue by BESPOKE3DPRINTING** [Ultimaker/Cura#230]
The latest version isn't allowing me to rotate, scale or mirror any of my .stl files
any ideas?
Kind regards

**Comment by nallath** [comment]
Please have a look at https://github.com/Ultimaker/Cura#logging-issues. We need more information before we can help.

**Comment by BagelOrb** [comment]
Besides the information @nallath is asking for:
do you get the rotate, scale etc. buttons on the bottom left?
Do you get the on-model arrows in red, green and blue?

**Comment by BESPOKE3DPRINTING** [comment]
I can see the buttons on the left but I am not seeing the on model co
ordinate system. It's like cura isn't recognising any of the stl files.

**Comment by awhiemstra** [comment]
Can you select objects by clicking on them?

**Comment by BESPOKE3DPRINTING** [comment]
No

**Comment by awhiemstra** [comment]
Then it is a duplicate of #84

**Comment by BESPOKE3DPRINTING** [comment]
Ok I'll try that.
Thanks for your support.
